src/systems/systems/src/hydraulic/update_iterator.rs

```diff
@@ -281,9 +275,14 @@ mod max_step_tests {
         for cur_time_step in &mut max_step {
             time_simulated += cur_time_step;
             actual_loop_num += 1;
+            println!(
+                "Loop done {} time step {}",
+                actual_loop_num,
+                cur_time_step.as_secs_f64()
+            );
```

Comment:
This log statement should probably not be commited?

Reply:
That's in unit tests only, but yeah not mandatory anyway